pages/components/card/card.css

```diff
@@ -0,0 +1,83 @@
+/* Importing external styles */
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+@import "../image/image.css";
+@import "../button/button.css";
+
+.vertical-card-wrapper {
+  width: 20rem;
+  border: 1px solid lightgray;
+}
+
+.add-btn,
+.wishlist-btn {
+  padding: 0.8rem;
+  width: 100%;
+  border-radius: 0;
+  font-size: calc(1.2 * var(--default-font-size));
+}
+
+.product-discount-price {
+  font-size: calc(1.3 * var(--default-font-size));
+  font-weight: 900;
+}
+
+.product-original-price {
+  font-size: calc(1.3 * var(--default-font-size));
+  text-decoration: line-through;
+}
+
+.product-discount {
+  font-weight: 900;
+  font-size: calc(1.3 * var(--default-font-size));
+  color: var(--success-dark-color);
+}
+
+.horizontal-card-wrapper {
+  width: 35rem;
+  border: 1px solid lightgray;
+}
+
+.vertical-card-badge-wrapper {
+  position: absolute;
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  top: 0;
+  left: 0;
+}
+
+.dismiss-badge-wrapper {
+  position: absolute;
+  font-size: calc(2.5 * var(--default-font-size));
+  background-color: var(--primary-color);
+  color: var(--white-color);
+  top: 5px;
+  right: 5px;
+  width: 1.8rem;
+  height: 1.8rem;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+.out-stock-wrapper {
+  position: absolute;
+  color: var(--white-color);
+  background-color: rgba(62, 67, 75, 0.5);
+  font-size: calc(3 * var(--default-font-size));
+  width: 20rem;
+  height: 32.28rem;
```

Comment:
avoid using height wifth together unless absolutely necessary, just a suggestion. Depends on the context

Reply:
I totally agree with you, about using height because of certain use cases.